server/db/model.py

```diff
@@ -28,6 +35,99 @@ class Meta:
         ignore_unknown_fields = True
 
 
+class Prospect(MongoModel):
+    owner = fields.ReferenceField("User")
+    email = fields.CharField()  # TODO can be primary key
+    first_name = fields.CharField()
+    last_name = fields.CharField()
+    status = fields.CharField()
+    campaigns: ...
+    last_contacted: ...
+    steps: ...
```

Comment:
I don't think we need campaigns, steps, last_contacted right now - so let's just remove it

Reply:
Removed the unused fields